src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py

```diff
@@ -1191,6 +1191,26 @@ def test_functionapp_with_app_insights_key(self, resource_group, storage_account
             'functionapp delete -g {} -n {}'.format(resource_group, functionapp_name))
 
 
+class FunctionAppWithAppInsightsConnString(ScenarioTest):
+    def __init__(self, method_name, config_file=None, recording_name=None, recording_processors=None, replay_processors=None, recording_patches=None, replay_patches=None, random_config_dir=False):
+        super().__init__(method_name, config_file, recording_name, recording_processors, replay_processors, recording_patches, replay_patches, random_config_dir)
+        self.cmd('extension add -n application-insights')
```

Comment:
`extension add -n` should not be here. It causes CI failure:
```
==================================== ERRORS ====================================
__________ ERROR collecting tests/latest/test_functionapp_commands.py __________
/usr/lib64/az/lib/python3.9/site-packages/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py:990: in __init__
    self.cmd('extension add -n application-insights')
/usr/lib64/az/lib/python3.9/site-packages/azure/cli/testsdk/base.py:176: in cmd
    return execute(self.cli_ctx, command, expect_failure=expect_failure).assert_with_checks(checks)
/usr/lib64/az/lib/python3.9/site-packages/azure/cli/testsdk/base.py:260: in __init__
    raise AssertionError('The command failed. Exit code: {}'.format(self.exit_code))
E   AssertionError: The command failed. Exit code: 1
__________ ERROR collecting tests/latest/test_functionapp_commands.py __________
/usr/lib64/az/lib/python3.9/site-packages/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py:1579: in __init__
    self.cmd('extension add -n application-insights')
/usr/lib64/az/lib/python3.9/site-packages/azure/cli/testsdk/base.py:176: in cmd
    return execute(self.cli_ctx, command, expect_failure=expect_failure).assert_with_checks(checks)
/usr/lib64/az/lib/python3.9/site-packages/azure/cli/testsdk/base.py:260: in __init__
    raise AssertionError('The command failed. Exit code: {}'.format(self.exit_code))
E   AssertionError: The command failed. Exit code: 1
__________ ERROR collecting tests/latest/test_functionapp_commands.py __________
/usr/lib64/az/lib/python3.9/site-packages/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py:1707: in __init__
    self.cmd('extension add -n application-insights')
/usr/lib64/az/lib/python3.9/site-packages/azure/cli/testsdk/base.py:176: in cmd
    return execute(self.cli_ctx, command, expect_failure=expect_failure).assert_with_checks(checks)
/usr/lib64/az/lib/python3.9/site-packages/azure/cli/testsdk/base.py:260: in __init__
    raise AssertionError('The command failed. Exit code: {}'.format(self.exit_code))
E   AssertionError: The command failed. Exit code: 1
---------- generated xml file: /azure_cli_test_result/appservice.xml -----------
=========================== short test summary info ============================
ERROR tests/latest/test_functionapp_commands.py::FunctionAppManagedEnvironment
ERROR tests/latest/test_functionapp_commands.py::FunctionAppWithAppInsightsConnString
ERROR tests/latest/test_functionapp_commands.py::FunctionappAppInsightsWorkspace

```

https://dev.azure.com/azclitools/public/_build/results?buildId=209680&view=logs&j=fce0fe10-dd1b-5814-f04b-ea25f95c49f0&t=f9fbf0f4-0b46-5841-08eb-fc21c02e7e0a&l=3124

Reply:
@kamperiadis Could you please take a look at this issue?